pymongo/encryption_options.py

```diff
@@ -219,3 +219,44 @@ def __init__(
         # Maps KMS provider name to a SSLContext.
         self._kms_ssl_contexts = _parse_kms_tls_options(kms_tls_options)
         self._bypass_query_analysis = bypass_query_analysis
+
+
+class EncryptionRangeOpts:
```

Comment:
Since EncryptionRangeOpts is always used with encrypt methods (ClientEncryption.encrypt/ClientEncryption.encryptExpression), can we rename it to "RangeOpts"?

Could we also rename the parameters from "range_options" -> "range_opts" to match the "opts" using in the class name?